force-app/main/default/lwc/donationHistoryTable/donationHistoryTable.js

```diff
@@ -1,17 +1,25 @@
 import { api, LightningElement, track, wire } from 'lwc';
-import {MOCK_DATA} from './donationHistoryTableData';
 import { getObjectInfo } from 'lightning/uiObjectInfoApi';
 import DATA_IMPORT from '@salesforce/schema/DataImport__c';
 import PAYMENT_FIELD from '@salesforce/schema/DataImport__c.Payment_Method__c';
 import donationHistoryDatatableAriaLabel from '@salesforce/label/c.donationHistoryDatatableAriaLabel';
 import RD2_ScheduleVisualizerColumnDate from '@salesforce/label/c.RD2_ScheduleVisualizerColumnDate';
+import getDonationHistory from '@salesforce/apex/DonationHistoryController.getDonationHistory';
 import commonAmount from '@salesforce/label/c.commonAmount';
 import donationHistoryDonorLabel from '@salesforce/label/c.donationHistoryDonorLabel';
 export default class DonationHistoryTable extends LightningElement {
     @api contactId;
 
+    recordsToLoad = 50;
```

Comment:
I think the records to load variable here is truly a constant, so lets bring it outside of the class and give it an UPPER_SNAKE_CASE name:

```suggestion
const RECORDS_TO_LOAD = 50;
export default class DonationHistoryTable extends LightningElement {
    @api contactId;
```

Reply:
changed to RECORDS_TO_LOAD constant